README.md

```diff
@@ -1,2 +1,37 @@
-# fantasy-land
-Specification for interoperability of common algebraic structures in PHP
+# FantasyLand [![Build Status](https://travis-ci.org/functional-php/fantasy-land.svg?branch=master)](https://travis-ci.org/functional-php/fantasy-land)
+## Introduction
+
+Specification for interoperability of common algebraic structures in PHP.
+The inspiration for this package is taken from fantastic project [fantasyland](https://github.com/fantasyland/fantasy-land) `[A]`.
```

Comment:
The inspiration for this package is the fantastic  [fantasyland](https://github.com/fantasyland/fantasy-land) project.